.github/workflows/publish-commit-bottles.yml

```diff
@@ -137,6 +139,13 @@ jobs:
             exit 1
           fi
 
+          # See https://github.com/octokit/octokit.net/issues/1763 for possible values.
+          if [[ "$mergeable_state" != "clean" ]] && [[ "$mergeable_state" != "unstable" ]]
+          then
+            echo "::error ::PR #$PR mergeable_state is $mergeable_state!"
+            exit 1
+          fi
+
```

Comment:
I think we'll need a retry mechanism here if `$mergeable_state` is `unknown`.

```console
$ for i in {0..3}; do gh api repos/Homebrew/homebrew-core/pulls/123791 -q .mergeable_state; sleep 1; done
unknown
blocked
blocked
blocked
```

Also, do we want to check for the PR's draft status?

Reply:
I moved the check into `automerge.yml` which already has a retry mechanism for querying the API.